pkg/build.ml

```diff
@@ -36,6 +36,9 @@ let () =
     Pkg.lib ~exts:[`Ext ".cmo"; `Ext ".cmx";`Ext ".cmi"; `Ext ".cmt"; `Ext ".cmxs"] "src/redoc_html";
     Pkg.lib ~exts:[`Ext ".cmo"; `Ext ".cmx";`Ext ".cmi"; `Ext ".cmt"] "vendor/cmdliner/cmdliner";
     Pkg.lib ~exts:[`Ext ".cmo"; `Ext ".cmx";`Ext ".cmi"; `Ext ".cmt"] "vendor/easy_format/easy_format";
+    Pkg.lib ~exts:[`Ext ".cmo"; `Ext ".cmx";`Ext ".cmi"; `Ext ".cmt"] "vendor/ppx_tools_versioned/ast_convenience_404";
+    Pkg.bin "vendor/ppx_tools_versioned/ppx_metaquot_404.ml" ~dst:"metaquot";
```

Comment:
Let's add a todo here? Hopefully we will avoid vendoring this when ppx_tools_versioned is available through any of the package manager.